pkg/utils/formatting.go

```diff
@@ -21,10 +22,22 @@ type ColumnConfig struct {
 	Alignment Alignment
 }
 
+func RuneWidth(s string) int {
```

Comment:
I was surprised that this performance improvement was so easy to implement, so I took a look at the runewidth repo and there are three open (stale) PRs that improve performance:

* https://github.com/mattn/go-runewidth/pull/78
* https://github.com/mattn/go-runewidth/pull/63
* https://github.com/mattn/go-runewidth/pull/42

Of those https://github.com/mattn/go-runewidth/pull/63 seems the most legit (It includes PR 42's approach, and was made by the guy who made tview).

I'm thinking we should use that fork, what are your thoughts? Seems the maintainer of the main repo is not very responsive.

If we do want to continue with our own function here, I think it ought to be called StringWidth given that it is returning the length of the string.

Reply:
Not sure what you mean by "that fork"; it's an unmerged PR. Are you suggesting to switch to the `unisegupdate` branch of rivo's fork? I'm not sure that's a good idea, because it could get rebased or deleted any time.

Reply:
Yep that's what I meant. My assumption was that rivo would be using that branch himself for something and so that it would be somewhat stable, but I take your point.

I've left a comment on the PR for both rivo and mattn: https://github.com/mattn/go-runewidth/pull/63

At any rate, I'm happy to merge this PR as-is, and we can keep an eye on that runewidth PR

Reply:
> If we do want to continue with our own function here, I think it ought to be called StringWidth given that it is returning the length of the string.

Makes sense, changed to StringWidth in 4935d2ff3316.